app/grandchallenge/components/serializers.py

```diff
@@ -148,23 +141,37 @@ def validate(self, attrs):
                     f"upload_session or image are required for interface "
                     f"kind {interface.kind}"
                 )
+        elif interface.requires_value:
+            if (
+                attrs.get("value") is None
+            ):  # Note: can also be False so check for None instead
+                raise serializers.ValidationError(
+                    f"value is required for interface "
+                    f"kind {interface.kind}"
+                )
+        elif interface.requires_file:
+            if not any(
+                [
+                    attrs.get("file"),
+                    attrs.get("user_upload"),
+                ]
+            ):
```

Comment:
Does this mean that we could now technically send over a `file` value, i.e. basically reuse file CIVs?

Reply:
I think using `file` is already currently possible: but it's the `file` field directly from the model. I **think** it entails that you could skip the creation of a UserUpload first.... but not sure. Haven't tested it: that path should already work now.